### ISSUE TITLE
Fatal error: Uncaught TypeError when creating product

### ISSUE THREAD

**Issue by 1337gut** [litecart/litecart#293]
### What where you trying to do?
Creating a new product in a roughly fresh installation

### What actually happened?
`Fatal error: Uncaught TypeError: implode(): Argument #2 ($array) must be of type ?array, string given in /home/ershop/public_html/adminhsv/catalog.app/edit_product.inc.php:12 Stack trace: #0 /home/ershop/public_html/adminhsv/catalog.app/edit_product.inc.php(12): implode() #1 /home/ershop/public_html/adminhsv/index.php(139): include('/home/ershop/pu...') #2 {main} thrown in /home/ershop/public_html/adminhsv/catalog.app/edit_product.inc.php on line 12`

### What did you expect to happen?
Well, not this.

### LiteCart Version

LiteCart 2.4.2

### PHP Version

PHP 7.3.33

### Error/Backtrace

```shell
[15-Jul-2022 19:54:41 Europe/Berlin] PHP Fatal error:  Uncaught TypeError: implode(): Argument #2 ($array) must be of type ?array, string given in /home/ershop/public_html/adminhsv/catalog.app/edit_product.inc.php:12
Stack trace:
#0 /home/ershop/public_html/adminhsv/catalog.app/edit_product.inc.php(12): implode()
#1 /home/ershop/public_html/adminhsv/index.php(139): include('/home/ershop/pu...')
#2 {main}
  thrown in /home/ershop/public_html/adminhsv/catalog.app/edit_product.inc.php on line 12
```

### Is this a problem that can be reproduced in the [demo platform](https://demo.litecart.net/admin)?

No

### If this problem could be related to a browser. Which one?

Not relevant

**Comment by timint** [comment]
> What did you expect to happen?
> Well, not this.
LOL

See if the problem persists with the 2.4.3 update.

**Comment by 1337gut** [comment]
Yeah, solved in 2.4.3
